### ISSUE TITLE
Authors not acknowledged?

### ISSUE THREAD

**Issue by Antti-L** [experimentarium/times#3]
I see the authors of Part I had been copied and put into `extra.md` in the root folder.
So, I would suggest to move this author list to the Part I folder (other parts have different authors).  I also suggest to remove "Antti Lettila" from that list (unknown name, looks like an error to me). 

And, if any changes have been made to the original document, it might be a good idea to acknowledge also the author of those changes.  :smile:

**Comment by olejandro** [comment]
Good point @Antti-L! As you have noticed I've started structuring the result of this experimental conversion a little bit. 😄 I am planning to include the info about the authors in markdown front matter. Good to keep this issue open until then.

**Comment by olejandro** [comment]
@Antti-L, does this look okay: https://times--89.org.readthedocs.build/en/89/
